### PR TITLE
Preview graph inside the modal.

### PR DESCRIPTION
**Pull request by cdricms**
# Now able to preview the graph inside the modal before inserting.

[plotfunction_preview_modal.webm](https://user-images.githubusercontent.com/36056008/184634821-2c03f09e-b994-406c-8984-4967a883ea26.webm)

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-}
\ No newline at end of file
+  "id": "obsidian-functionplot",
+  "name": "Obsidian Functionplot",
+  "minAppVersion": "0.11.13",
+  "description": "A plugin for displaying mathematical graphs in obsidian.md.",
+  "author": "leonhma",
+  "authorUrl": "https://github.com/leonhma",
+  "isDesktopOnly": false,
+  "version": "1.1.1"
```
Comment (leonhma) [inline manifest.json:9]: you dont even have to bump the version yourself, the release workflow will take care of that

File: versions.json [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
-	"1.0.1": "0.11.13",
-	"1.0.2": "0.11.13",
-	"1.0.3": "0.11.13",
-	"1.1.0": "0.11.13"
+  "1.0.0": "0.11.13",
+  "1.0.1": "0.11.13",
+  "1.0.2": "0.11.13",
+  "1.0.3": "0.11.13",
+  "1.1.0": "0.11.13",
+  "1.1.1": "0.11.13"
```
Comment (leonhma) [inline versions.json:7]: again all taken care of by the release workflow

**Review by leonhma — COMMENTED** [review]
Code looks good to me, I just need to test it. Will merge it sometime later today.

File: manifest.json (hunk 1 above)
Comment (cdricms) [reply manifest.json]: Good to know, it's been taken care of.

**Comment by leonhma** [comment]
Just FYI, everything version-related is managed by the release workflow and is set to the tag of the release, so changing any values in there doesn't affect the final release. The version fields (apart from minVersion in the manifest) are basically just placeholders.

**Comment by cdricms** [comment]
Okay, thank you for the information !

**Comment by leonhma** [comment]
Just tested it. Works quite nicely!

**Pull request merged**